docs/zh-CN/flink/configuration/transform-plugins/Replace.md

```diff
@@ -0,0 +1,94 @@
+# Replace
```

Comment:
There is no need to provide Chinese documentation, We will find a tool to help generate the Chinese documentation, we have already discussed in this [mail thread ](https://lists.apache.org/thread/j1vo1yccqp79bv69bzokbqq3n58zytql)

Reply:
ok, I remove it!